Bust Config.rb

```diff
@@ -167,6 +199,13 @@ def custom_bust_display_options
         nil,
         face_name = 'Vera emo'
       ]
+    elsif Busty::TOPLESS_SIMON_MAP[$game_message.face_name] && $game_actors[2].character_index == 2
+      # Simon is using his almost naked sprite, override with topless faceset
```

Comment:
As it would allow to reduce the total number of files and length of the config, and we're already hacking the world here, I'm tempted to make a sheet (of one face) for topless Simon the young and go for an additional nested if there:
```rb
if $game_message.face_index == 7 && ["face002b", "face002b dark eyes"].include?($game_message.face_index)
  return [
      nil,
      nil,
      face_name = 'Simon topless young emo',
      face_index  = 0,
  ]
end
```
Thus allowing to remove the one difference between `1 Simon topless dark eyes` and `face002b topless dark eyes`, and thus merge them. Same for the other two sheets were young Simon throws a wrench in. Also allow to get rid of the bit in SUBSET_TO_BUST.

Reply:
I deliberately made the faceset files completely interchangeable between the clothed and non-clothed versions, so that Sierra can switch between them with the minimum of effort if necessary. I don't think it's worthwhile doing additional gymnastics just to make the config a little more concise.